eng/tools/summarize-impact/src/cli.ts

```diff
@@ -1,29 +1,15 @@
-#!/usr/bin/env node
-
 import { getChangedFilesStatuses } from "@azure-tools/specs-shared/changed-files";
 import { setOutput } from "@azure-tools/specs-shared/error-reporting";
 import { evaluateImpact, getRPaaSFolderList } from "./impact.js";
 
 import { getRootFolder } from "@azure-tools/specs-shared/simple-git";
 import { Octokit } from "@octokit/rest";
-import fs from "fs";
+import { writeFile } from "fs/promises";
 import { parseArgs, ParseArgsConfig } from "node:util";
-import { join } from "path";
+import { resolve } from "path";
 import { LabelContext } from "./labelling-types.js";
 import { PRContext } from "./PRContext.js";
 
-export async function getRoot(inputPath: string): Promise<string> {
```

Comment:
We no longer wrap calls to getRootFolder(), but I think the default behavior when throwing an unhandled exception will be good enough.